src/createStore.js

```diff
@@ -1,8 +1,11 @@
 import { createStore as originalCreateStore } from 'redux'
 
 export function createStore (reducer) {
-  const extension = window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
-  return originalCreateStore(reducer, extension)
+  let enhancer
+  if (process.env.NODE_ENV !== 'production' && window.__REDUX_DEVTOOLS_EXTENSION__) {
+    enhancer = window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
Fair enough as long as we don't measure with them enabled. I don't think anybody optimized Redux DevTools for perf.